utils/contracts.ts

```diff
@@ -158,3 +182,16 @@ export const getFIFOMarket = async ({
     hre,
     signer,
   });
+
+export const getScheduleTestHarness = async ({
```

Comment:
I also had to add this to satisfy the compiler in `getContractsFromDeployments` which was expecting a value for `ScheduleTestHarness` in addition to the other contracts